README.md

```diff
@@ -1,70 +1,48 @@
-* Provides a style API to enable abstracting the internal structure of a component so that it can be styled externally. For example, you can style the label inside a button, or style the play button of a video player from outside these components.
-* Uses mixins and variants so, for example, you can tell the same component to use a Wix theme or a Bootstrap theme.
-
+* Provides a style API to enable abstracting the internal structure of a component so that it can be styled externally. 
+* Uses themes so you can apply a different look and feel to the same components across your web application.
+* Enables you to create, reuse and style pseudo-classes and pseudo-elements.
 At build time, the preprocessor converts the **Stylable** CSS into flat, static, valid vanilla CSS that works cross-browser.
 
-Viva CSS, and welcome **Stylable**. We hope you like it. Learn how to use it with step by step instructions and code examples.
-
-## Components
-
-[Stylable-components](https://github.com/wix/stylable-components) - react stylable components with mobx
-
-## Guides
-
-* [Install & Configure Stylable](./docs/guides/install-configure.md)
-
-* [Component Basics](./docs/guides/component-basics.md)
-
-* [Stylable Application](./guides/stylable-application.md)
-
-* [Stylable Component Library](./guides/stylable-component-library.md)
-
-## References
-
-* [Imports](./docs/references/imports.md)
-
-* [Root](./docs/references/root.md)
-
-* [Class Selectors](./docs/references/class-selectors.md)
+Learn more in our [Documentation Center]().
+Or read our story [here](../usefulInfo/overview.md) in GitHub.
 
-* [Tag Selectors](./docs/references/tag-selectors.md)
+## Code Example
 
-* [Extend stylesheet](./docs/references/extend-stylesheet.md)
+*Ido to add really cool code example*
 
-* [Pseudo-Classes](./docs/references/pseudo-classes.md)
+## Installation
 
-* [Pseudo-Elements](./docs/references/pseudo-elements.md)
+Install **Stylable** as a dependency in your local project.
 
-* [Global selectors](./docs/references/global-selectors.md)
+Using npm:
+```bash
+npm install stylable --save
+```
+Using yarn:
+```bash
+yarn add stylable
+```
 
-* [Compose CSS Class](./docs/references/compose-css-class.md)
+While the **stylable** package includes a programmatic API and can be used directly, we strongly suggest using
+the **stylable-integration** package that powers and enables using the `.st.css` files in your project's build cycle:
+https://github.com/wixplosives/stylable-integration
 
-* [Mixins](./docs/references/mixin-syntax.md)
+## Component Library
 
-* [Variables](./docs/references/variables.md)
+**Stylable** includes a rich [component library](https://github.com/wix/stylable-components) for out-of-the-box use.
 
-* [Theme](./references/theme.md)
+## Contributing
 
-* [Namespace](./docs/references/namespace.md)
+Read our [contributing guidelines]() for details on our code of conduct, and the process for submitting pull requests. *Need guideline doc from Bruce*
```

Comment:
open a document called CONTRIBUTING.md and in it write something like "coming soon..."